src/public_pool/tor_public_pool.sh

```diff
@@ -1,34 +1,39 @@
 function enable_tor_public_pool {
-if [[ $OS == Mac ]] ; then no_mac ; return 1 ; fi
+    if [[ $OS == Mac ]]; then no_mac; return 1; fi
 
-enable_tor_general || return 1
+    enable_tor_general || return 1
 
-if sudo grep "HiddenServiceDir $varlibtor/public_pool-service/" \
-    $torrc | grep -v "^#" >$dn 2>&1 ; then true ; debug "true" ; else debug "else"
-    echo "HiddenServiceDir $varlibtor/public_pool-service/" | sudo tee -a $torrc >$dn 2>&1
+    if sudo grep "HiddenServiceDir $varlibtor/public_pool-service/" $torrc | grep -v "^#" >$dn 2>&1; then
+        true
+        debug "true"
+    else
+        debug "else"
+        echo "HiddenServiceDir $varlibtor/public_pool-service/" | sudo tee -a $torrc >$dn 2>&1
     fi
 
-if sudo grep "HiddenServicePort 5055 127.0.0.1:5052" \
-    $torrc | grep -v "^#" >$dn 2>&1 ; then true ; else
-    echo "HiddenServicePort 5055 127.0.0.1:5052" | sudo tee -a $torrc >$dn 2>&1
+    if sudo grep "HiddenServicePort 5055 127.0.0.1:5052" $torrc | grep -v "^#" >$dn 2>&1; then
```

Comment:
Now that I look, it's probably better to have no else and put the code under "then" by adding a "!" before sudo